### ISSUE TITLE
Append DSL Issue : Adds text in between instead of at the end (gwen-web-2.29.0)

### ISSUE THREAD

**Issue by Rahul9844** [gwen-interpreter/gwen-web#65]
I am using below sequence of steps

```
And  I execute javascript "$("div.sidebar-container i.fa-search").closest(".filter").find("input").val("Item Allow Recalculation Of Amounts Flag")"

And  I append " " to search attribute textbox on data format
```

Currently the append dsl adds space in between. In earlier gwen version (2.26.5), it use to work correctly, adding space at the end of the text.

**Comment by Rahul9844** [comment]
![image](https://user-images.githubusercontent.com/34029569/49355596-5c8a0a80-f6ee-11e8-9551-7352b3bca0dd.png)

**Comment by bjuric** [comment]
Fixed in release 2.29.1
